tests/test_imports.py

```diff
@@ -29,8 +29,8 @@ def test_activity_mapper(gold_mapper):
     gold_mapper.activity_mapper()
     gold_mapper.post_nmdc_database_object()
     db = gold_mapper.get_database_object_dump()
-    assert len(db.metagenome_annotation_activity_set) == 1
-    assert len(db.mags_activity_set) == 1
+    #TODO I think this assert statement should be 2 ANC
```

Comment:
@mbthornton-lbl  where you able to double check the expected number here?

Reply:
No, and I don't really understand the import code well enough yet to know what sensible expect values should be.
In this case if we use the prod `imports.yaml` we get different results: 4 vs. 2

I would prefer to deal with imports outside of the scope of this PR
https://github.com/microbiomedata/nmdc_automation/issues/254